doc/source/echoharmonogram_pl.md

```diff
@@ -0,0 +1,33 @@
+
+## Configuration via configuration.yaml
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: ecoharmonogram_pl
+      args:
+        town_input: town_input
+        street_input: street_input
+        house_number_input: house_number_input
+```
+
+## Example
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: ecoharmonogram_pl
+      args:
+        town_input: Krzeszowice
+        street_input: Wyki
+        house_number_input: 
+```
+
+## Keep in mind
+Ecoharmonogram data is filled by each city, so that some towns doesn't require street name while some does. 
```

Comment:
What should a user enter as street name if it not required? Could you make an optional argument out of it?

Reply:
If it will be null it will work just fine.
I would rather like users to fill all the fields as it will give them a guarantee that the integration works.
Also If they will leave for example street name empty the integration will work fine. Also worth mention that leaving street name empty is absolutely valid as some villages doesn't have streets names just a house numbers -

Reply:
`null` means that the user has to enter the variable name, but leave the value empty, right? I think this is not what most users have in mind. I think they want to omit the variable name also, this is what I call an optional argument. Other sources also do it that way.

Reply:
ok, applied the fix